etc/start_faux

```diff
@@ -2,7 +2,7 @@
 
 echo Running faux $*
 
-umount /etc/resolv.conf
+umount /etc/resolv.conf # Necessary to run dhclient in docker containers
```

Comment:
comment goes on previous line not trailing (just style convention)

Reply:
Resolved.